### PR TITLE
Change icons (landing banner)

### PR DESCRIPTION
**Pull request by nknguyenhc**

### CODE REVIEW

File: new/src/assets/images/landing.svg
Comment (Singa-pirate) [inline new/src/assets/images/landing.svg:1]: Can consider exporting from figma as png / try different svg format.

The current svg does not render clearly on iPhone chrome:
svg_iphone_chrome

For png:
png_iphone_chrome

**Review by Singa-pirate — CHANGES_REQUESTED** [review]
LGTM except for the blurry svg issue on iPhone

File: new/src/assets/images/landing.svg
Comment (nknguyenhc) [reply new/src/assets/images/landing.svg:1]: Changed to PNG

**Review by Singa-pirate — APPROVED** [review]
LGTM!

**Pull request merged**